scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -230,6 +237,31 @@ boolean pauseJob(JobId jobId) {
         }
     }
 
+    boolean updateStartAt(JobId jobId, String startAt) {
```

Comment:
It could be moved yes. I will create a pull request with the refactored new class